tests/core/multiprocess/test_runner.py

```diff
@@ -34,6 +37,8 @@ class AbstractMultiprocessRunnerTest(BasicRotestUnitTest):
 
     fixtures = ['case_ut.json']
 
+    @mock.patch("rotest.management.client.client.Requester",
```

Comment:
found out why you need this...
the clients in the worker she be mock clients

Reply:
? how

Reply:
solved together